op-tests/test-bls-ops.txt

```diff
@@ -67,8 +67,8 @@ g1_negate 0 => FAIL
 g1_negate 0xb7f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bb 0xb7f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bb => FAIL
 g1_negate => FAIL
 g1_negate ( 1 2 3 ) => FAIL
-; The input doesn't have to be a valid G1 point
-g1_negate 0xb7f1d3a7319092346345638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bb => 0x97f1d3a7319092346345638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bb | 1396
+; The input has to be a valid G1 point
```

Comment:
How did you find this? Trial & error?

Reply:
you mean how I found a 48 byte value that's not a valid G1 point?
I believe I took a valid one and corrupted it some. I would expect *most* 48-byte values are not valid G1 points, at least I can't recall ever producing one by chance.